runtimes/CMakeLists.txt

```diff
@@ -24,7 +24,7 @@ list(INSERT CMAKE_MODULE_PATH 0
 # We order libraries to mirror roughly how they are layered, except that compiler-rt can depend
 # on libc++, so we put it after.
 set(LLVM_DEFAULT_RUNTIMES "libc;libunwind;libcxxabi;pstl;libcxx;compiler-rt;openmp;offload")
-set(LLVM_SUPPORTED_RUNTIMES "${LLVM_DEFAULT_RUNTIMES};llvm-libgcc")
+set(LLVM_SUPPORTED_RUNTIMES "${LLVM_DEFAULT_RUNTIMES};llvm-libgcc;flang-rt")
```

Comment:
Any reason why `flang-rt` shouldn't be built by default?

Reply:
flang is not part of the default build, so `LLVM_ENABLE_PROJECTS=all LLVM_ENABLE_RUNTIMES=all` would yield an error.

Reply:
> flang is not part of the default build, so `LLVM_ENABLE_PROJECTS=all LLVM_ENABLE_RUNTIMES=all` would yield an error.

AFAIU there have been long-standing discussions to include flang into the default build, and this was mostly waiting for the `flang-new` -> `flang` rename. After I couldn't find an issue for this, I opened one: #112789